net/openvswitch/patches/0100-netdev-linux-Use-unsigned-int-for-ifi_flags.patch

```diff
@@ -24,5 +24,5 @@ index c0471be15..0750e5f2c 100644
  
      old_flags = netdev->ifi_flags;
 -- 
-2.16.2
+2.16.3
```

Comment:
nitpick: this change looks redundant; if doing this, maybe remove the git version altogether, so that this does not happen again

curios: did you run `make package/openvswitch/refresh` to update/refresh the patches ? or did you do this manually ?

Reply:
I used git-format-patch.  Will add options to remove the version.

Reply:
maybe try `make package/openvswitch/refresh` and see what does :)